README.md

```diff
@@ -151,6 +151,7 @@ Key                | Type   | Allowed                                  | Default
 `windowMode`       | String | `'normal'`, `'maximized'`, `'minimized'` | `'normal'`                   | How the window should be displayed by default
 `hotkey`           | String | Any string                               | None                         | A global hotkey to associate to opening this shortcut, like `'CTRL+ALT+F'`
 `workingDirectory` | String | Any valid path to a folder               | None                         | The working directory for the shortcut when it launches
+`VBScriptPath`     | String | Path to external VBS file                | None                         | This is an advanced option specifically for projects packaged with `pkg`. You can create a copy of the `windows.vbs` file outside of your packaged executable and pass in the location of it. If your `vbs` file differs in any way to the version shipped with this library, it have bugs. Ensure you are programmatically copying the shipped version, so if it changes in a future update, your code will still work. If you are not using `pkg`, you should not include `VBScriptPath` in your Windows settings.
```

Comment:
```suggestion
`VBScriptPath`     | String | Path to external VBS file                | None                         | This is an advanced option specifically for projects packaged with `pkg`. You can create a copy of the `windows.vbs` file outside of your packaged executable and pass in the location of it. If your `vbs` file differs in any way to the version shipped with this library, it will have bugs. Ensure you are programmatically copying the shipped version, so if it changes in a future update, your code will still work. If you are not using `pkg`, you should not include `VBScriptPath` in your Windows settings.
```